canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -230,12 +234,11 @@ $palette-border-color: $ui-03;
 }
 
 .palette-flyout-category-text {
-	line-height: 46px;  /* Must be same as containing div height */
-	overflow: hidden;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	display: flex;
+	width: 100%;
+	overflow-wrap: break-word;
+	line-height: 1.3;
 	margin-left: 16px;
-	width: 135px; // We need to allow enough space for search numbers to appear eg "(12)"
 }
```

Comment:
It still doesn't look the same. 

<img width="257" alt="image" src="https://user-images.githubusercontent.com/15114439/140170301-5cef46a8-da1d-48a1-810e-ce3829ea7162.png">

Also you have changed the hover behavior which need  to stay the same. The hover behavior now is strange ... 

![PalDrag](https://user-images.githubusercontent.com/15114439/140171036-af24c025-fdbb-42da-8c5a-340f689e0452.gif)